package.json

```diff
@@ -1,81 +1,84 @@
-      ".eslintrc.js"
-    ],
-    "testURL": "http://localhost/"
-  },
-  "author": {
-    "name": "Iliyan Peychev"
-  },
-  "license": "LGPL-3.0",
-  "keywords": [
-    "Liferay",
-    "AMD",
-    "ES6",
-    "Loader"
-  ],
-  "bugs": {
-    "url": "https://github.com/liferay/liferay-amd-loader/issues"
-  },
-  "homepage": "https://github.com/liferay/liferay-amd-loader",
-  "files": [
-    ".babelrc",
-    ".eslintrc.js",
-    ".publishrc",
-    "LICENSE.md",
-    "package.json",
-    "README.md",
-    "webpack.config.js",
-    "src",
-    "bin",
-    "build",
-    "src"
-  ],
-  "dependencies": {
-    "es6-promise": "^4.0.5"
-  },
-  "devDependencies": {
-    "babel-loader": "^7.1.2",
-    "babel-preset-es2015": "^6.24.1",
-    "combohandler": "^0.4.0",
-    "eslint": "^5.15.1",
-    "eslint-config-liferay": "^3.0.0",
-    "fs-extra": "^5.0.0",
-    "globby": "^7.1.1",
-    "http-server": "^0.11.1",
-    "jest": "^22.2.1",
-    "prettier": "^1.16.4",
-    "publish-please": "^2.3.1",
-    "uglifyjs-webpack-plugin": "^1.2.0",
-    "webpack": "^3.10.0"
-  }
+	"name": "liferay-amd-loader",
+	"version": "4.0.0-alpha.3",
+	"description": "AMD Loader with support for combo URL and conditional loading",
+	"scripts": {
+		"clean": "rm -rf build",
+		"build": "node bin/build-loader.js",
+		"ci": "npm run format:check && npm run lint && npm run build && npm run build-demo && npm test",
+		"build-demo": "node bin/build-demo.js",
+		"demo": "node bin/run-demo.js",
+		"test": "jest --runInBand",
+		"format": "prettier --write '{*,.*}.js' 'bin/**/{*,.*}.js' 'src/**/{*,.*}.js'",
+		"format:check": "prettier --list-different '{*,.*}.js' 'bin/**/{*,.*}.js' 'src/**/{*,.*}.js'",
+		"lint": "eslint '*.js' 'bin/**/{*,.*}.js' 'src/**/{*,.*}.js'",
+		"lint:fix": "eslint --fix '*.js' 'bin/**/{*,.*}.js' 'src/**/{*,.*}.js'",
+		"prepublish": "publish-please guard",
+		"publish-please": "publish-please",
+		"prepublishOnly": "npm run build && npm run build-demo && npm test",
+		"proxyPortal": "webpack-dev-server --config webpack.proxyPortal.js"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/liferay/liferay-amd-loader.git"
+	},
+	"jest": {
+		"collectCoverage": true,
+		"coverageDirectory": "build/coverage",
+		"modulePathIgnorePatterns": [
+			"liferay-amd-loader/build/.*",
+			".*/__tests__/__fixtures__/.*"
+		],
+		"testPathIgnorePatterns": [
+			".eslintrc.js"
+		],
+		"testURL": "http://localhost/"
+	},
+	"author": {
+		"name": "Iliyan Peychev"
+	},
+	"license": "LGPL-3.0",
+	"keywords": [
+		"Liferay",
+		"AMD",
+		"ES6",
+		"Loader"
+	],
+	"bugs": {
+		"url": "https://github.com/liferay/liferay-amd-loader/issues"
+	},
+	"homepage": "https://github.com/liferay/liferay-amd-loader",
+	"files": [
+		".babelrc",
+		".eslintrc.js",
+		".publishrc",
+		"LICENSE.md",
+		"package.json",
+		"README.md",
+		"webpack.config.js",
+		"src",
+		"bin",
+		"build",
+		"src"
+	],
+	"dependencies": {
+		"es6-promise": "^4.0.5"
```

Comment:
I know this has been here for a long time, but it stood up right now since the file has been formatted. 

I think we could remove the promise polyfill right now and make this the responsability of calling code. That way, in DXP, for example, we can make sure we only load it on IE11 reducing the size of the loader considerably. That is, unless we're using something like cancellablepromise here... 🤔 

What do you think @wincent, @izaera?

Reply:
We are using the resolvable promise which is somewhat exotic, but should work on top of the standard Promise. However, if we remove the polyfill we need to make sure nothing breaks...

Reply:
That sounds reasonable to me. In other projects I've worked with the requirement was to "bring your own polyfill" for the very basic stuff, like Promises, if you needed it.

Reply:
I've removed it in #203